llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -811,6 +823,8 @@ class VPRecipeWithIRFlags : public VPSingleDefRecipe {
 
   FastMathFlags getFastMathFlags() const;
 
+  bool isNonNeg() const { return NonNegFlags.NonNeg; }
```

Comment:
Can we assert that OpType == OperationType::NonNegOp here, like for other accessors? otherwise we would access a non-active union element.